Multisig/UI/Safe Management/Add Safe/EnterSafeNameViewController.swift

```diff
@@ -48,7 +48,16 @@ class EnterSafeNameViewController: UIViewController {
     @objc private func didTapNextButton() {
         guard let name = name, let address = address else { return }
         Safe.create(address: address.checksummed, name: name)
-        completion()
+        if let _ = App.shared.settings.signingKeyAddress {
+            completion()
+        } else if AppSettings.hasShownImportKeyOnboarding() {
```

Comment:
I'm confused... why do we need to show the Enter seed phrase screen after entering safe name?

Reply:
You are right